radix-engine-common/src/constants/transaction_execution.rs

```diff
@@ -57,6 +57,9 @@ pub const MAX_METADATA_KEY_STRING_LEN: usize = 100;
 /// The max SBOR size of metadata value
 pub const MAX_METADATA_VALUE_SBOR_LEN: usize = 512;
 
+/// The max depth to support, per recursive move-to-store operation.
```

Comment:
To clarify, this doesn't put bounds on the actual max persisted depth right? If for example, I call move-to-store on a leaf node and then call move-to-store on the new leaf node.

Reply:
If so, can we rename to `MAX_RECURSIVE_ON_PERSIST_DEPTH` to make this clear?